src/debug.c

```diff
@@ -990,6 +994,17 @@ void debugCommand(client *c) {
             return;
         }
         addReply(c, shared.ok);
+    } else if (!strcasecmp(c->argv[1]->ptr, "sleep-after-fork-seconds") && c->argc == 3) {
+        double sleep_after_fork_seconds;
+        if (getDoubleFromObjectOrReply(c, c->argv[2], &sleep_after_fork_seconds, NULL) != C_OK) {
+            addReply(c, shared.err);
+            return;
+        }
+        server.debug_sleep_after_fork_ms = (int)(sleep_after_fork_seconds * 1e6);
```

Comment:
That's not milliseconds. 1000 ms = 1 second.

It's microseconds (µs).

Ideas:


* debug_usleep_after_fork
* debug_sleep_after_fork_us